_api_app/app/Sites/SitesDataService.php

```diff
@@ -166,6 +169,16 @@ public function create($cloneFrom = null)
         $this->array2xmlFile(['site' => $sites], $this->XML_FILE, $this->ROOT_ELEMENT);
         $site['order'] = count($sites) - 1;
 
+        $dbPath = $dir . '/shop-db.sqlite';
+        if (file_exists($dbPath)) {
```

Comment:
Check plugin availability like it's done here: https://github.com/berta-cms/berta/blob/master/_api_app/app/Sites/Sections/Entries/SectionEntriesController.php#L159